<?xml version="1.0"?>
<launch>

  <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}"/>

  <node pkg="ariac_group1" 
        type="ariac_group1_kitting_arm_node" 
        name="kitting_arm" output="screen"></node>

 <node pkg="ariac_group1" 
        type="ariac_group1_gantry_arm_node" 
        name="gantry_arm" output="screen"></node>


</launch>
